src/exercises_sources/friday/manifolds.lean

```diff
@@ -521,11 +521,11 @@ to use the library
 section you_should_probably_skip_this
 
 /- If `M` is a manifold modelled on a vector space `E`, then the underlying type for the tangent
-bundle is just `Σ (x : M), tangent_space x M` (i.e., the disjoint union of the tangent spaces,
+bundle is effectively `Σ (x : M), tangent_space x M` (i.e., the disjoint union of the tangent spaces,
```

Comment:
@fpvandoorn, do you think this needs slightly more care, or is this comment fine?

Reply:
This has changed since LFTCM 2020, and I think this change is fine.